packager/react-packager/src/SocketInterface/index.js

```diff
@@ -35,10 +35,15 @@ const SocketInterface = {
         }
       });
 
-      const sockPath = path.join(
+      let sockPath = path.join(
         tmpdir,
         'react-packager-' + hash.digest('hex')
       );
+      if (process.platform==='win32'){
+        sockPath = sockPath.replace(/^\//, '')
+        sockPath = sockPath.replace(/\//g, '-')
```

Comment:
Are forward slashes not allowed in paths on Windows? Consider adding a short comment explaining these two lines.